### DISCUSSION TITLE
Whole techmino website not working

### DISCUSSION THREAD

**Discussion by lunauii** [26F-Studio/Techmino#618] [General]
I cannot use the website nor validate an account, im currently using iOS, so i believe it probably has something to do with webkit?

[a comment by an automated account is not shown]

**Comment by ParticleG** [comment]
The domain `home.techmino.org` would be available again after March

**Reply by lidawei97688** [reply]
有没有考虑过像《我的世界》那样允许玩家搭建私人服务器？

**Reply by C29H25N3O5** [reply]
> 有没有考虑过像《我的世界》那样允许玩家搭建私人服务器？


Z酱貌似不太喜欢这个主意(
Apparently, MrZ is not a big fan of this idea, so...

**Reply by ParticleG** [reply]
> 有没有考虑过像《我的世界》那样允许玩家搭建私人服务器？

未来可以使用新的后端源码搭建完整私服，但是需要自己设置并部署数据库。除此之外也可以通过认证将你的个人服务器加入转发节点

[a comment by an automated account is not shown]
